libs/utils/android/workloads/jankbench.py

```diff
@@ -154,6 +154,8 @@ def run(self, exp_dir, test_name, iterations, collect=''):
 
         # Switch back to screen auto rotation
         Screen.set_orientation(self.target, auto=True)
+        
```

Comment:
That's a good one! :+1: 
Can you remove this empty line and change the previous comment to be more "generic", e.g. "Reset initial setup"?